Ginger/GingerCore/Actions/ActHandleBrowserAlert.cs

```diff
@@ -87,7 +87,7 @@ public override String ActionType
 
         bool IObsoleteAction.IsObsoleteForPlatform(ePlatformType platform)
         {
-            if (platform == ePlatformType.Web || platform == ePlatformType.Mobile || platform == ePlatformType.NA)
+            if (platform == ePlatformType.Web || platform == ePlatformType.NA)
             {
```

Comment:
i asked to remove the Mobile only for GenericElement Action and leave it for rest